eq-author/src/App/page/Design/answers/withDeleteOption.js

```diff
@@ -31,6 +31,7 @@ export const mapMutateToProps = ({ mutate }) => ({
     return mutate({
       variables: { input: option },
       update,
+      refetchQueries: ["GetPage"],
```

Comment:
Couldn't `deleteOption` just refetch the validations for the other options instead of refetching the whole page?

Reply:
We've done this in some other places where deleting the entity returns the parent which means the above manual cache management could be removed as well

Reply:
Refactored so deleteOption returns parent answer and removed the refetchQuery option